requirements/edx/stanford.txt

```diff
@@ -18,4 +18,4 @@ ubcpi-xblock==0.6.5
 -e git+https://github.com/mfogel/django-settings-context-processor.git#egg=settings_context_processor
 -e git+https://github.com/open-craft/problem-builder.git@v2.6.5#egg=xblock-problem-builder==2.6.5
 -e git+https://github.com/Stanford-Online/xblock_3d_viewer.git@b1d6cf90d0909decfae51b7a34fa8baac3666d96#egg=mt3d-xblock
-git+https://github.com/Stanford-Online/xblock-platform-tour.git@v1.0.4#egg=xblock-platform-tour==1.0.4
+-e git+https://github.com/Stanford-Online/xblock-platform-tour.git@v1.0.4#egg=xblock-platform-tour==1.0.4
```

Comment:
Why are you changing this to use `-e`?

Reply:
cargo cult, the other xblocks have it...

Reply:
I think my preference would be to not change it here in this PR (considering it "off-topic") _yet_, since we're doing it inconsistently elsewhere. Ultimately, we should update everything to do it "one true way".
FWIW: edx has given conflicting advice on how to use this throughout the years.